go.mod

```diff
@@ -0,0 +1,5 @@
+module main
```

Comment:
Please rename your package according to your repo URL. Also, give it a proper name, because main doesn't; tell me what your project does at first glance.

For example, let's say your project is named `deliverableOne`, you should name it as `module github.com/jorgenino/deliverableOne`.